nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl.bat

```diff
@@ -207,6 +202,14 @@ if "%JAVA_VERSION%" lss "%REQUIRED_JAVA_VERSION%" (
   timeout /t 30
   goto END
 )
+set JAVA_VERSION_TOOLS=9
+if "%JAVA_VERSION%" geq "%JAVA_VERSION_TOOLS%" (
```

Comment:
greater or equal than; didn't you want to do the opposite lesser than?

Reply:
Yes, but I fall onto an issue when realizing it, the comparison is made on strings, even on java required version.
It implies if required is 9 and actual is 11, it will not work.
I'm adapting the script